src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml

```diff
@@ -129,7 +129,7 @@
 		        Value="{StaticResource MaterialFabPadding}" />
 
 		<!--  Workaround for WinUI issue: https://github.com/microsoft/microsoft-ui-xaml/issues/6388  -->
-		<Setter Property="um:ControlExtensions.Icon"
+		<Setter Property="um:ControlExtensions.LeadingIcon"
```

Comment:
Ah, I don't like this.

I forgot that ControlExtensions.Icon is used in controls other than PasswordBox and TextBox where it may not always make sense to rename it to `LeadingIcon`

Perhaps we leave ControlExtensions.Icon as it is to avoid the breaking change and introduce something like `TextBoxExtensions.LeadingIcon/TrailingIcon`?

@Xiaoy312 / @agneszitte thoughts?

Reply:
In my opinion I think it will be better also and less confusing

Reply:
So it would be:

### TextBoxExtensions

`LeadingIcon`
`IsLeadingIconVisible`
`LeadingCommand`

`TrailingIcon`
`IsTrailingIconVisible`
`TrailingCommand`

And ControlExtensions would be as it was, remain untouched, right?

Reply:
@Xiaoy312 / @agneszitte does that sound good to you? ^

Reply:
Also, considering that we already have the `Helpers > ControlExtensions` menu option in the **Uno.Themes SampleApp**, which showcases the TextBox icon, should we add a new menu option called TextBoxExtensions? We can then move the TextBox Leading Icon there and also include the Trailing Icon.

@kazo0 @Xiaoy312 @agneszitte